LoopKit/Persistence/Model.xcdatamodeld/Modelv2.xcdatamodel/contents

```diff
@@ -32,6 +32,7 @@
         <attribute name="uploadState" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="NO"/>
         <attribute name="uuid" attributeType="UUID" usesScalarValueType="NO"/>
         <attribute name="value" attributeType="Double" usesScalarValueType="YES"/>
+        <attribute name="wasUserEntered" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES"/>
```

Comment:
Assuming CoreData will migrate using this default value, do we want it to start with "NO"?  Maybe it should init to whatever `!isDisplayOnly` is?  Or...?  WDYT?

Reply:
Hmm, good question. 

Technically it should be set to the *same* as `isDisplayOnly`. For any existing records it will set to `NO` (the default), but technically should match whatever `isDisplayOnly` is set to. 

Do you know an easy way to do that here? Or does it require an explicit migration? If it requires a migration, then I don't think it is worth the effort since:
- it is only going to affect DIY folks *once* after they upgrade to this code
- DIY only keeps glucose values around for 24 hours
- DIY folks aren't likely going to have more than a couple calibrations in that 24 hour period (if any)
- no code will use `isDisplayOnly=true` and `wasUserEntered=false` given current CGM devices
- so it won't have any actual effect

Reply:
The only way I know how to do it is via manual migration.  I'm not fully versed in the subtleties, but it sounds to me like the risk is extremely small if there's an anomaly due to migration, as you say.   But it was worth asking :)

Reply:
I believe you could do this with a mapping model.